File: PRAKTIKUM 3/PRAK304_2310817220028_SheilaSabina.c
#include <stdio.h>

int main() {
    int angka;
    
    printf("Input\n");
    scanf("%d", &angka);

    if (angka < 0 || angka >= 100) {
        printf("Anda Menginput Melebihi Limit Bilangan\n");
    } else if (angka == 0) {
        printf("Nol\n");
    } else {
        int satuan = angka % 10;
        int puluhan = angka / 10;

        if (puluhan > 1) {
            
            printf("Puluhan");
            if (satuan > 0) {
                printf(" %d", satuan);
            }
        } else if (puluhan == 1) {
            
            if (satuan == 1) {
                printf("Sebelas");
            } else if (satuan == 0) {
                printf("Sepuluh");
            } else {
                printf("Belasan");
            }
        } else {

            if (satuan == 1) {
                printf("Satuan");
            } else {
                printf("Satuan");
            }
        }
        printf("\n");
    }

    return 0;
}

File: PRAKTIKUM 3/PRAK304_2310817220028_SheilaSabina.py
angka = int(input('Input\n'))

if angka == 0:
    print("\nOutput\nNol")
elif angka > 0 and angka < 10:
    print("\nOutput\nSatuan")
elif angka >= 10 and angka < 20:
    print("\nOutput\nBelasan")
elif angka >= 20 and angka < 100:
    print("\nOutput\nPuluhan")
else:
    print("\nOutput\nAnda Menginput Melebihi Limit Bilangan")